### ISSUE TITLE
custom delimiters

### ISSUE THREAD

**Issue by tunnckoCore** [jonschlinkert/dry#1]
specifically because the opening delimiter.

**Comment by jonschlinkert** [comment]
yeah, it's already (unofficially) supported. You'll need to look at the code to see how lexers and compilers are registered though. we can add official support in the future

**Comment by jonschlinkert** [comment]
Wow, I can't believe how long it's been. Well, I'm not sure if this will be supported, but I think we can probably make it happen. I'll look into it before we release: https://github.com/jonschlinkert/dry/pull/3
